rpc/block_svc_routines.c

```diff
@@ -4200,6 +4200,7 @@ block_replace_1_svc_st(blockReplace *blk, struct svc_req *rqstp)
 
   if (GB_ALLOC_N(reply->out, 8192) < 0) {
     GB_FREE(reply);
+    reply = NULL;
```

Comment:
@lxbsz 
```
void                                                                             
gbFree(void *ptrptr)                                                             
{                                                                                
  int save_errno = errno;                                                        
                                                                                 
                                                                                 
  if(*(void**)ptrptr == NULL) {                                                  
   return;                                                                       
  }                                                                              
                                                                                 
  free(*(void**)ptrptr);                                                         
  *(void**)ptrptr = NULL;                                                        
  errno = save_errno;                                                                                                 
}    

```                 

as part of GB_FREE it is supposed to assign NULL to reply, is anything wrong in above function ?

Reply:
Okay, I missed this part.
Thanks.